.github/ISSUE_TEMPLATE/infrastructure.yaml

```diff
@@ -0,0 +1,24 @@
+name: Infrastructure
+description: Improve CI/CD for the project
+labels:
+  - infra
+body:
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: |
+        What would you like to be added for CI/CD?
+        Please provide as much info as possible
+    validations:
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+        
```

Comment:
![image](https://github.com/ajktown/api/assets/53258958/f9a541ba-f533-48a7-98d0-491c28f5d003)
?

Reply:
Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)